README.md

```diff
@@ -1,24 +1,57 @@
-# README
+# Budget-App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- Is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby on Rails
 
-* System dependencies
+### Video Link
+- Here is a link to the video explaining the project [link](https://www.loom.com/share/bcb5bbd8f88848afb995e0b694c26583)
 
-* Configuration
+## Live Demo
 
-* Database creation
+No live demo available
 
-* Database initialization
+### Download it as a ZIP file
 
-* How to run the test suite
+or
 
-* Services (job queues, cache servers, search engines, etc.)
+### Clone it in your local machine using GIT
+```
+Clone repo: https://github.com/kwekubright/expenses_app.git
+```
 
-* Deployment instructions
+then
 
-* ...
+```run
+
+- bundle install
+- rails db:migrate
+- rails server
+
+```
+### Live
+Link [link](https://6353a7ef5b4a5e5f60b2452d--beautiful-rugelach-0b2089.netlify.app/)
```

Comment:
Your live demo link is not working.

![image](https://user-images.githubusercontent.com/38088818/197332172-d6ba5339-16f7-4c62-8f88-db38616bef52.png)

- Kindly make sure that your app is deployed properly and that the live demo link works as expected. Please try to deploy with Heroku for now since it has server support and you can [read this article](https://microverse.zendesk.com/hc/en-us/articles/10490872573715) about what Microverse is doing about the [discontinuing of Heroku’s free tier](https://blog.heroku.com/next-chapter) after November 28th, 2022.